cloud/drift/status.go

```diff
@@ -85,6 +105,50 @@ func (s Status) String() string {
 		return "drifted"
 	case Failed:
 		return "failed"
+	default:
+		return "unrecognized (" + strconv.Itoa(int(s)) + ")"
+	}
+}
+
+// Is tells if status matches the provided filter.
+func (s Status) Is(filter FilterStatus) bool {
+	return FilterStatus(s)&filter != 0
+}
+
+// NewStatusFilter creates a new filter for drift statuses.
+func NewStatusFilter(str string) (FilterStatus, error) {
+	if str == "unhealthy" {
+		return UnhealthyFilter, nil
+	} else if str == "healthy" {
+		return HealthyFilter, nil
+	}
+
+	s := NewStatus(str)
+	if s == Unrecognized {
+		return FilterStatus(0), errors.E("unrecognized drift status filter: %s", str)
```

Comment:
The parsing error for the different filter flags is slightly inconsistent in it's wording:
```
% terramate list --drift-status=foobar
Error: failed to parse drift filter
> unrecognized drift status filter: foobar

% terramate list --status=foobar       
Error: unrecognized stack filter
> unrecognized status filter: foobar

% terramate list --deployment-status=foobar
Error: failed to parse deployment filter
> unrecognized deployment status filter: foobar
```